recipes/h5utils/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "h5utils" %}
+{% set version = "1.13.1" %}
+{% set sha256 = "c5a76f064d6daa3e65583dce2b61202510e67cf6590f076af9a8aa72511d7d65" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/stevengj/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libpng >=1.6.32,<1.6.35
+    - hdf5 1.10.1
+  run:
+    - libpng >=1.6.32,<1.6.35
+    - hdf5 1.10.1
+
+test:
+  commands:
+    - ${PREFIX}/bin/h5topng -h
```

Comment:
Also please add tests (help or otherwise) for...

* `h5totxt`
* `h5fromtxt`
* `h5tovtk`